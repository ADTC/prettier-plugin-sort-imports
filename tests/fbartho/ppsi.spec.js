const testSpecificConfig = require('./prettier.config.js')

run_spec(__dirname, ['typescript'], testSpecificConfig);
