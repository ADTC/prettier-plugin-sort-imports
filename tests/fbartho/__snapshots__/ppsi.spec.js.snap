// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fbartho-issue16.ts - typescript-verify: fbartho-issue16.ts 1`] = `
/* eslint-disable @typescript-eslint/no-unused-vars */
/* eslint-disable @typescript-eslint/naming-convention */

import { DateTime } from "luxon";
import { Inject, Service } from "typedi";

import EmailService from "./email";
import {
	CareNotesDatabase,
	OfficeStaffDatabase,
	ShiftsDatabase,
	OfficeReport,
	OfficeStaffContactInfo,
	OfficeStaffRoles,
	AgencyConfigDatabase,
} from "../database";

import { EmailRecipient, Shift, TriggerProp } from "../types";
import SMSService from "./sms";
import { logger } from "@hca/logger";
import MESSAGES, { MESSAGE_TRIGGER_PROPS, MESSAGE_TYPES } from "../constants/messages";
import snakecase from "lodash.snakecase";

export interface OfficeReportMessageData {
	careNoteDocId: string;
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/* eslint-disable @typescript-eslint/no-unused-vars */

/* eslint-disable @typescript-eslint/naming-convention */
import snakecase from "lodash.snakecase";
import { DateTime } from "luxon";
import { Inject, Service } from "typedi";

import { logger } from "@hca/logger";

import MESSAGES, { MESSAGE_TRIGGER_PROPS, MESSAGE_TYPES } from "../constants/messages";
import {
	AgencyConfigDatabase,
	CareNotesDatabase,
	OfficeReport,
	OfficeStaffContactInfo,
	OfficeStaffDatabase,
	OfficeStaffRoles,
	ShiftsDatabase,
} from "../database";
import { EmailRecipient, Shift, TriggerProp } from "../types";
import EmailService from "./email";
import SMSService from "./sms";

export interface OfficeReportMessageData {
	careNoteDocId: string;
}

`;

exports[`prettier.config.js - typescript-verify: prettier.config.js 1`] = `
module.exports = {
	printWidth: 100,
	semi: true,
	singleQuote: false, // Single quotes are common in text-strings
	trailingComma: "all", // Improves refactoring / minimizes git-conflicts
	tabWidth: 2,
	useTabs: true,
	// @ianvs/prettier-plugin-sort-imports
	importOrder: ["^@hca", "^[.]+"],
	importOrderBuiltinModulesToTop: true,
	importOrderCaseInsensitive: true,
	importOrderParserPlugins: ["typescript", "jsx", "decorators-legacy"],
	importOrderSeparation: true,
	importOrderSortSpecifiers: true,
};
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
module.exports = {
	printWidth: 100,
	semi: true,
	singleQuote: false, // Single quotes are common in text-strings
	trailingComma: "all", // Improves refactoring / minimizes git-conflicts
	tabWidth: 2,
	useTabs: true,
	// @ianvs/prettier-plugin-sort-imports
	importOrder: ["^@hca", "^[.]+"],
	importOrderBuiltinModulesToTop: true,
	importOrderCaseInsensitive: true,
	importOrderParserPlugins: ["typescript", "jsx", "decorators-legacy"],
	importOrderSeparation: true,
	importOrderSortSpecifiers: true,
};

`;
